__tests__/lib/string.test.ts

```diff
@@ -151,7 +155,32 @@ describe('ago', () => {
     new Date().setMinutes(new Date().getMinutes() - 10),
     new Date().setHours(new Date().getHours() - 1),
     new Date().setDate(new Date().getDate() - 1),
+    new Date().setDate(new Date().getDate() - 20),
```

Comment:
Don't need to change in this PR, I realize it was already set up like this, but good tip for tests: each of these should be written as an `it` clause, with a description, so when one breaks we know exactly which failed. Written like this, one failure fails the whole thing, and determining which failed is harder.

Reply:
Also it's hard to review bc idk which lines up with which :P